src/domain/synthetics/accountStats/useAccountVolumeStats.ts

```diff
@@ -0,0 +1,95 @@
+import { gql } from "@apollo/client";
+import { getSubsquidGraphClient } from "lib/subgraph";
+import { useMemo } from "react";
+import useSWR from "swr";
+import { subDays, format, eachDayOfInterval } from "date-fns";
+import { toUtcDayStart } from "lib/dates";
+import { CONFIG_UPDATE_INTERVAL } from "lib/timeConstants";
+import { ARBITRUM, AVALANCHE } from "config/chains";
+
+const LARGE_ACCOUNT_CHAINS = [ARBITRUM, AVALANCHE];
```

Comment:
this is just SUPPORTED_CHAIN_IDS

Reply:
The difference is that we don't like to collect volume data stats for AVALANCHE_FUJI